tests/cli/test_cmds_logs.py

```diff
@@ -388,6 +388,19 @@ def test_extract_utc_timestamp_from_log_line_when_invalid_date_format():
     assert not logs.extract_utc_timestamp_from_log_line(line)
 
 
+def test_paasta_log_line_passes_filter_true_with_nested_container():
+    service = "fake_service"
+    clusters = ["fake_cluster1"]
+    instance = "fake_instance.container"
+
+    with mock.patch(
+        "paasta_tools.cli.utils.verify_instances", autospec=True
+    ) as verify_instances_patch:
+        verify_instances_patch.return_value = True
+
+        assert verify_instances_patch(instance, service, clusters)
```

Comment:
This doesn't actually test anything -- you've replaced `verify_instances` with a mock object that will always return `True`, so you could run `assert verify_intances_patch("abc", 123, False)` and it'd still return True.

I think you should patch `verify_instances` and then call `paasta_logs` and assert that `verify_instances` was called with just the instance names without the suffxies.  Let me know if you want help doing that.